components/ai_chat/resources/page/components/main/style.module.scss

```diff
@@ -68,10 +85,11 @@
 }
 
 .logoTitle {
+  font-family: Poppins, sans-serif;
```

Comment:
Should be a Nala variable?

Reply:
<img width="564" alt="image" src="https://github.com/brave/brave-core/assets/8665427/51d4b6f6-8c43-4748-982c-d7fc4b8c1e86">

nala font variables come with line height and font sizes defined. i could still use them and do an override? i dunno.

Reply:
@aguscruiz is this not one of our standard font sizes? I guess we can override just font-size if not @nullhook

Reply:
I have a style created for it in the browser tokens file. But I think that file has never been merged into the actual code you have for Nala

<img width="650" alt="image" src="https://github.com/brave/brave-core/assets/15734630/709d5415-22ed-45f2-b4d3-d3f50707219b">

Reply:
i have updated to use a nala token and then override